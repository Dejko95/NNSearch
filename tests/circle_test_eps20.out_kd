Max visited: 106
Avg visited: 88.27
