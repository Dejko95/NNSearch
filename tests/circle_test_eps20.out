
Max visited: 35
Avg visited: 26.78
Max error: 0.002266979935930058
Avg err: 3.3127615254318287E-4
Precision: 56 %
